### ISSUE TITLE
There are missing files

### ISSUE THREAD

**Issue by Outeger** [Sfresneda/SolarSystem-ARKit#1]
There are missing folders and files. So when I build it, there is only camera opening..

<img width="262" alt="ekran resmi 2017-07-20 01 17 43" src="https://user-images.githubusercontent.com/2598906/28392130-5461ea70-6ce9-11e7-9daf-82150b379e9f.png">

**Comment by Sfresneda** [comment]
@Outeger yep is right, I had a little issue with my MBP and I can't upload the complete project currently. I think I can do it this week.
Thanks for report

**Comment by Outeger** [comment]
I glad to know when it is really working. Thanks for your fast reply.

**Comment by johndpope** [comment]
fyi - https://github.com/ThumbWorks/AugmentedSolarSystem

I'm curious to know what the wireframe view does. I'm working through this ticket to add a spherical sphere. https://github.com/ThumbWorks/AugmentedSolarSystem/issues/46
